webapp/graphite/settings.py

```diff
@@ -85,6 +85,7 @@
 LOG_ROTATION = True
 LOG_ROTATION_COUNT = 1
 MAX_FETCH_RETRIES = 2
+MAX_FOUND_METRICS = 0
```

Comment:
Note: you may want to implement METRIC_FIND_FAILURE_THRESHOLD and METRICS_FIND_WARNING_THRESHOLD in the place that calls all the finders.